src/ui/App.jsx

```diff
@@ -124,17 +135,87 @@ class App extends Component {
     cookies.remove('jwt', { path: '/' });
   };
 
-  exploreMappingsByOrganism = (organimsId) => {
-    this.setState(
-      {
-        exploreMappingsByOrganism: organimsId,
-      },
-      () => {
+  exploreMappingsByOrganism = (organism) => {
+    const { activeFacets } = this.state;
+    const activeFacetsCopy = {
+      ...activeFacets,
+      organism,
+    }
+
+    this.setState({
+      activeFacets: activeFacetsCopy,
+    }, () => {
         this.exploreMappingsAction();
```

Comment:
There is an issue in `exploreMappingsAction`, as all filters should be reset when one of the "Explore" buttons is clicked.